src/components/organisms/CreateProposalModal.vue

```diff
@@ -112,7 +112,9 @@ export default {
             this.isDisabled = false
             this.enableAllFields()
             this.clearAllFieldValues()
-            this.$refs.form.resetValidation()
+
+            // TODO I belive we don't need this
+            //this.$refs.form.resetValidation() 
```

Comment:
I believe this simply removes any input field highlighting that might remain even after the values have been programatically reset. It would need to be tested with and without the line active to confirm.